nodejs/aws-infra/network.ts

```diff
@@ -101,8 +101,10 @@ export class Network extends pulumi.ComponentResource implements ClusterNetworkA
         if (!defaultNetwork) {
             const vpc = aws.ec2.getVpc({default: true});
             const vpcId = vpc.then(v => v.id);
-            const subnetIds = aws.ec2.getSubnetIds({ vpcId: vpcId }).then(subnets => subnets.ids);
-            const defaultSecurityGroup = aws.ec2.getSecurityGroup({ name: "default", vpcId: vpcId }).then(sg => sg.id);
+            const subnetIds = vpcId.then(id => aws.ec2.getSubnetIds({ vpcId: id })).then(subnets => subnets.ids);
+            const defaultSecurityGroup = vpcId.then(id => aws.ec2.getSecurityGroup(
+                { name: "default", vpcId: id },
+            )).then(sg => sg.id);
```

Comment:
nit: why'd you split this line, but not the above one as well?

Reply:
It was necessary because this line spread past the 120 column limit enforced by ts-lint.